_pages/tools/windrider.md

```diff
@@ -7,7 +7,7 @@ breadcrumb: tests
 
 # Windrider
 
-**Windrider has been decommissioned on 01/17/2013. The source code will be still available at the link provided below.**
+Windrider has been decommissioned on 01/17/2013. The source code will be still available at the link provided below.
```

Comment:
Sorry, should've pointed this one out, good catch.

The linter identified having an entire paragraph bolded as an issue, stating it would be best practice to use a heading level instead of bolding an entire paragraph.  

I notice that on the /tools/pathload2/ page, it also has a similar "decomissioned" statement, but it is an h2.  Do you think it would be better to just change this to an h2 so it is consistent with the other pages where they have a decommissioned sentence?